crates/zed/src/languages/hcl/config.toml

```
@@ -0,0 +1,13 @@
+name = "HCL"
+path_suffixes = ["hcl", "hcl2"]
```

```suggestion
path_suffixes = ["hcl"]
```

I think it's safe to remove the `.hcl2` extension here. The [vscode-hcl extension doesn't claim it](https://github.com/hashicorp/vscode-hcl/blob/0ca34e6ce7dae6269858e39f687a2ac3732871ed/package.json#L47-L71) and it doesn't seem to be widely used.

---

Hah, old-er habits die hard. I can't remember the last time I encountered a hcl2 file to be fair. Good spot